version.h.in

```diff
@@ -1,3 +1,8 @@
 #pragma once
 
 #define @UPPERCASE_PROJECT_NAME@_VERSION "@PROJECT_VERSION@"
+
+#define @UPPERCASE_PROJECT_NAME@_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
+#define @UPPERCASE_PROJECT_NAME@_VERSION_MINOR @PROJECT_VERSION_MINOR@
+#define @UPPERCASE_PROJECT_NAME@_VERSION_PATCH @PROJECT_VERSION_PATCH@
+#define @UPPERCASE_PROJECT_NAME@_VERSION_BUILD @PROJECT_VERSION_TWEAK@
```

Comment:
Could you add a newline at the end of the file?

Reply:
Yep